src/modules/item/Item.tsx

```diff
@@ -456,6 +457,9 @@ const FolderContent = ({
   // render each children recursively
   return (
     <>
+      <Helmet>
+        <title>{item.name}</title>
```

Comment:
I think we did not discuss whether we would like to somehow mention the root item name in the page title. 
Lets say that you are currently in a Math course and you are viewing the page for the third exercise. If we only show the current page item name it would be: `Exercise 3`
If we add the root item name it could be: `Exercise 3 | Mathematics 2024` 

I think it could give a bit more context to what is currently being displayed. What do you think @mariembencheikh @pyphilia ?